From a06969b83a8515b2d32d6d4e9d65555b75229b5f Mon Sep 17 00:00:00 2001
From: Alaa Hleihel <ahleihel@redhat.com>
Date: Tue, 12 May 2020 10:55:28 -0400
Subject: [PATCH 209/312] [netdrv] mlx5: Update list of unsupported devices

Message-id: <20200512105530.4207-123-ahleihel@redhat.com>
Patchwork-id: 306995
Patchwork-instance: patchwork
O-Subject: [RHEL8.3 BZ 1789382 122/124] mlx5: Update list of unsupported devices
Bugzilla: 1823685 1789382
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Kamal Heib <kheib@redhat.com>
RH-Acked-by: Jarod Wilson <jarod@redhat.com>

Bugzilla: http://bugzilla.redhat.com/1789382
Bugzilla: http://bugzilla.redhat.com/1823685
Upstream: RHEL-only

Mark the new devices as unsupported so customers know that these devices
have not passed full RHEL testing.
Marking the devices as unsupported doesn't taint the kernel, which marking
them as tech-preview would do.
This enables the devices to go through qualification & certification
testing post RHEL-8.3 GA.

The new devices are:
 - ConnectX-6 LX
 - ConnectX-7
 - BlueField-2

Signed-off-by: Alaa Hleihel <ahleihel@redhat.com>
Signed-off-by: Frantisek Hrbata <fhrbata@redhat.com>
---
 drivers/net/ethernet/mellanox/mlx5/core/main.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/net/ethernet/mellanox/mlx5/core/main.c b/drivers/net/ethernet/mellanox/mlx5/core/main.c
index 05d66dd6791b..314d6d95cf76 100644
--- a/drivers/net/ethernet/mellanox/mlx5/core/main.c
+++ b/drivers/net/ethernet/mellanox/mlx5/core/main.c
@@ -738,6 +738,9 @@ static int mlx5_core_set_issi(struct mlx5_core_dev *dev)
 static const struct pci_device_id mlx5_core_hw_unsupp_pci_table[] = {
 	{ PCI_VDEVICE(MELLANOX, 0x101d) },			/* ConnectX-6 Dx */
 	{ PCI_VDEVICE(MELLANOX, 0x101e), MLX5_PCI_DEV_IS_VF},	/* ConnectX Family mlx5Gen Virtual Function */
+	{ PCI_VDEVICE(MELLANOX, 0x101f) },			/* ConnectX-6 LX */
+	{ PCI_VDEVICE(MELLANOX, 0x1021) },			/* ConnectX-7 */
+	{ PCI_VDEVICE(MELLANOX, 0xa2d6) },			/* BlueField-2 integrated ConnectX-6 Dx network controller */
 	{ 0, }
 };
 
-- 
2.13.6

